tests/helpers/_assertions.py

```diff
@@ -6,7 +6,15 @@
 from safeds.data.tabular.containers import Cell, Column, Table
 
 
-def assert_tables_equal(table1: Table, table2: Table) -> None:
+def assert_tables_equal(
+    table1: Table,
+    table2: Table,
+    *,
+    ignore_column_order: bool = False,
+    ignore_row_order: bool = False,
+    ignore_types: bool = False,
+    check_exact: bool = False,
```

Comment:
Should either all be `ignore` or `check`. Having some options that enable and some that disable strict checking is confusing.

Reply:
Changed check_exact to ignore_float_imprecision and made it True by default